### PR TITLE
button lift and text shadow

### LINKED ISSUE #14: Improve design in any way
This issue is a general invitation to anyone to submit pull requests that improve the design of the site in any way.

Tiny incremental improvements are best: find a design tweak that makes the site 1-5% better, and submit a PR for it. That will allow us to review your changes and merge them in quickly.

### PR DESCRIPTION
**Pull request by tof-tof**
added button lift on hover
added extra shadow to buttons to help it stand out from the cards
set the tracking on the buttons to be a bit wider as the text as upper case
added background shadow to directionary text

resolve #14

### CODE REVIEW

**Comment by RedFox0x20** [comment]
Please note the linter issues were previously fixed in a couple commits that your branch is behind on. It's important to pull the latest commits before creating a PR. :)

**Comment by audreyfeldroy** [comment]
I love what you did here @tof-tof! These sorts of subtle style and UX improvements to delight the visitor are wonderful.

Thank you @RedFox0x20 for providing friendly guidance here. I appreciate you pitching in on the review side so much!

Yep, the best time to pull commits from the `main` branch is before the PR. The next best time is after you submit the PR and realize your branch is behind...nice work pulling and updating your PR, and fixing the lint error 🙂 

I'm in meetings for awhile, but I'll follow up later with a more detailed technical review of your diff.

File: pages/_app.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import 'tailwindcss/tailwind.css';
+// import 'tailwindcss/tailwind.css';
```
Comment (audreyfeldroy) [inline pages/_app.js:1]:
```suggestion
```

**Comment by audreyfeldroy** [comment]
The diff looks good. I'm still getting the hang of Tailwind CSS and am a bit tired, so I'd actually like to revisit the rest of this review tomorrow with fresh eyes.

@marekrozmus if you'd like, you're welcome to review and merge this before I return tomorrow.

File: pages/_app.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+import '../styles/global.css';
```
Comment (marekrozmus) [inline pages/_app.js:2]: Please group the local imports below the external modules like:
```javascript
import PropTypes from 'prop-types';
import React from 'react';

import '../styles/global.css';
```

File: styles/global.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+@layer components {
+  .btn {
+    @apply tracking-wider shadow-2xl transform transition hover:-translate-y-0.5;  
+  }
+
+  .shaded-text{
+    @apply bg-black bg-opacity-10 py-2 rounded-lg;
+  }
+
```
Comment (marekrozmus) [inline styles/global.css:13]: Please remove empty line

**Review by marekrozmus — CHANGES_REQUESTED** [review]
@audreyfeldroy I'm not huge fan of lifting buttons as we are changing here default behaviour and look and feel of `daisyui` but if you like it then go ahead 🙂

**Comment by audreyfeldroy** [comment]
Thanks for the feedback about changing default daisyUI behavior @marekrozmus. It's a good point. Custom behavior does increase the maintenance effort on our side. I've given it serious consideration. My read of daisyUI is that it's like Bootstrap, where default components are provided but it's common/expected to customize the styles. https://daisyui.com/docs/customize looks like it encourages customization.

Here the lift and shadow behavior adds enough improvement that I'd like to accept this PR. It's also subtle enough that I see it as compatible with #76.

I do think it's the sort of change that could potentially be useful upstream to daisyUI. @tof-tof before I recommend that you submit an upstream PR: 

@saadeghi Hey, would you be interested in a daisyUI PR for the button lift and/or text shadow in this PR? See it in action in the Netlify deploy preview above.

**Review by pydanny — APPROVED** [review]
Looks good to me.

**Comment by pydanny** [comment]
@tof-tof Awesome PR! 👍🏻

**Comment by saadeghi** [comment]
@audreyfeldroy I encourage everyone to customize design decisions for their projects. Thanks to Tailwind's utility classes it's easy to customize things.

**Pull request merged**